pkgs/top-level/python-packages.nix

```
@@ -4721,6 +4721,8 @@ in {
 
   prometheus_client = callPackage ../development/python-modules/prometheus_client { };
 
+  prometheus_flask_exporter = callPackage ../development/python-modules/prometheus_flask_exporter { };
```

```suggestion
  prometheus-flask-exporter = callPackage ../development/python-modules/prometheus-flask-exporter { };
```

We adhere to the PEP replaces _ with -.